cellengine/utils/api_client/APIClient.py

```diff
@@ -119,6 +144,69 @@ def update_entity(self, experiment_id, _id, entity_type, body) -> dict:
             json=body,
         )
 
+    def _get_path(self, entity: CE) -> str:
+        fullpath = f"{self.base_url}/{entity.path}"
+        return fullpath
+
+    def post_and_structure(self, _class, path, body, **kwargs):
+        res = self._post(f"{self.base_url}/{path}", json=body, params=kwargs)
+        return converter.structure(res, _class)
+
+    def unstructure_and_clean(self, entity):
+        (cls, path, body) = (
+            entity.__class__,
+            entity.path,
+            converter.unstructure(entity),
+        )
+        if body["_id"] == "None" or body["_id"] is None:
+            del body["_id"]  # https://github.com/primitybio/cellengine/issues/5800
+        return (cls, path, body)
+
+    # fmt: off
+    # temporary fix for https://github.com/psf/black/issues/1797
+    @overload
+    def create(self, entity: Attachment, **kwargs) -> Attachment: ...
+    @overload
+    def create(self, entity: Compensation, **kwargs) -> Compensation: ...
+    @overload
+    def create(self, entity: Experiment, **kwargs) -> Experiment: ...
+    @overload
+    def create(self, entity: FcsFile, **kwargs) -> FcsFile: ...
+    @overload
+    def create(self, entity: Gate, **kwargs) -> Gate: ...
+    @overload
+    def create(self, entity: Population, **kwargs) -> Population: ...
+    @overload
+    def create(self, entity: ScaleSet, **kwargs) -> ScaleSet: ...
+    # fmt: on
+
+    def create(
+        self, entity, **kwargs
+    ) -> Union[
+        Attachment,
+        Compensation,
+        Experiment,
+        FcsFile,
+        Gate,
+        Population,
+        ScaleSet,
+    ]:
```

Comment:
Can this be shortened to this?
```suggestion
    ) -> Union[CE, List[CE]]:
```

Reply:
simplified :+1: